interfaces/IOCDM.h

```diff
@@ -296,8 +296,14 @@ class EXTERNAL KeyId {
 public:
     inline bool IsValid() const
     {
-        static const KeyId InvalidKey;
```

Comment:
Just remove the static, but leave the comparison logic in the KeyId class..

Reply:
What I meant to say is, leave the comparison logic in the KeyId class (there is for example a work-a-round in there that you now do not apply. I think it is sufficient to only remove the static on the InvalidKey Declaration. So this megthod should be:
inline bool IsValid() const {
    const KeyId InvalidKey;
    return (operator!=(InvalidKey));
}

Reply:
@pwielders This pull request has same as describe above. So the method is as follows:

    inline bool IsValid() const
    {
        const KeyId InvalidKey;
        return (operator!=(InvalidKey));
    }